var/spack/repos/builtin/packages/py-nltk/package.py

```diff
@@ -13,11 +13,28 @@ class PyNltk(PythonPackage):
 
     version('3.5', sha256='845365449cd8c5f9731f7cb9f8bd6fd0767553b9d53af9eb1b3abf7700936b35')
 
+    variant('data', default=False, description='Download the NLTK data')
+
     depends_on('python@3.5:', type=('build', 'run'))
-    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools', type=('build', 'run'))
     depends_on('py-click', type=('build', 'run'))
```

Comment:
It looks like we're also missing a dependency on `py-joblib`.

Reply:
added